use-cases/using-SRIOV-and-kata.md

```diff
@@ -0,0 +1,313 @@
+# Setup to use SR-IOV with Kata Containers
```

Comment:
Not essential for this PR given it's a rework of an existing doc, but... it would be great if you could add a table of contents here as it is very difficult to navigate long docs like this when the different headings are rendered in such a similarly sized font.